blocks/init/src/blocks/custom/featured-posts/components/featured-posts-options.js

```diff
@@ -0,0 +1,67 @@
+import React from 'react'; // eslint-disable-line no-unused-vars
+import { __ } from '@wordpress/i18n';
+import { PanelBody, SelectControl, FormTokenField, RangeControl } from '@wordpress/components';
+
+export const postTypeOptions = [
+  { label: __('Posts', 'eightshift-blocks'), value: 'posts' },
+];
+
+export const FeaturedPostsOptions = (props) => {
+  const {
```

Comment:
spread this props like in this example https://github.com/infinum/eightshift-frontend-libs/blob/develop/blocks/init/src/blocks/custom/card/components/card-editor.js